runtime/autoload/zip.vim

```diff
@@ -131,8 +131,7 @@ fun! zip#Browse(zipfile)
  \                '" Select a file with cursor and press ENTER'])
   keepj $
 
-"  call Decho("exe silent r! ".g:zip_unzipcmd." -l -- ".s:Escape(a:zipfile,1))
-  exe "keepj sil! r! ".g:zip_unzipcmd." -Z -1 -- ".s:Escape(a:zipfile,1)
+  exe $"keepj sil! r! {g:zip_unzipcmd} -Z1 -- {s:Escape(a:zipfile, 1)}"
```

Comment:
can you please change the `:sil!` to `:sil` so that error messages are properly reported? I suppose this is also hiding the error message on Windows when `shellslash` option is set.